.infrastructure/98_variables.tf

```diff
@@ -24,4 +24,16 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+### required for Strapi Site on AWS
+variable "dns_domain_name" {
+  description = "DNS domain for the B2B Portal product"
+  type        = string
+}
+
+variable "use_custom_certificate" {
```

Comment:
ok, but I think we should start with the value of `use_custom_certificate` equals to `false` until we have decided the DNS for our product

Reply:
Yes